conda_forge_tick/update_recipe/version.py

```diff
@@ -152,20 +153,36 @@ def _try_pypi_api(url_tmpl: str, context: MutableMapping, hash_type: str, cmeta:
         return None, None
```

Comment:
Not your fault, but can we get a docstring for `_try_pypi_api` now that we modified it? I personally don't understand the functionality from the function name.

Reply:
Done